kubectl-plugin/pkg/cmd/cluster/get_test.go

```diff
@@ -0,0 +1,195 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gosuri/uitable"
```

Comment:
Is this library commonly used? The repository does not seem to be actively maintained https://github.com/gosuri/uitable.

Reply:
I don't think its commonly used. I was looking into using [this](https://pkg.go.dev/k8s.io/cli-runtime/pkg/printers) instead, which should be more reliable and more commonly used.

Reply:
@kevin85421  Should I make the change in this PR or make a separate PR?